website/static/css/stoplight-base.css

```diff
@@ -0,0 +1,111 @@
+/*
+    DO NOT EDIT - any customizations should be made in stoplight-custom.css
```

Comment:
Note that this css file is ONLY loaded on API docs pages. The only piece of the site other than the API docs that displays is the nav bar. I confirmed that everything in the nav bar looks as expected